encrypted-config-value-bundle-dropwizard1/src/main/java/com/palantir/config/crypto/EncryptConfigValueCommand.java

```diff
@@ -60,6 +61,6 @@ public void run(Bootstrap<?> bootstrap, Namespace namespace) throws Exception {
         EncryptedValue encryptedValue = algorithm.getEncryptedValue(value, keyWithAlgorithm);
 
         // print the resulting encrypted value to the console
-        System.out.println(encryptedValue);
+        System.out.println(encryptedValue.serialize());
```

Comment:
OK -- I don't feel super strongly about this (requiring a call to `serialize()` is more semantically consistent), but was just a thought.